project_generation/helpers.go

```diff
@@ -374,13 +379,15 @@ func OptionPromptInput(ctx context.Context, prompt string, options ...string) (s
 	return options[optionSelected], nil
 }
 
-// InitGoModules will initialise the go modules for a project at a given directory
+// InitGoModules will initialise the go modules for a project at a given directory unless go.mod already exists
 func InitGoModules(ctx context.Context, pathToRepo, name string) {
-	cmd := exec.Command("go", "mod", "init", "github.com/ONSdigital/"+name)
-	cmd.Dir = pathToRepo
-	err := cmd.Run()
-	if err != nil {
-		log.Event(ctx, "error initialising go modules", log.Error(err))
+	if _, err := os.Stat(pathToRepo + "/go.mod"); os.IsNotExist(err) {
```

Comment:
It's slightly more complicated than that. An existing go.mod will not throw an error. There is improvement to be made here and I will do so but it it will take the form of…
* go.mod exists -> do nothing
* go.mod cannot be stat'ed (return error)
* go.mod does not exist -> carry on and make it

Reply:
Thanks for taking a look 👍 even if it means there's 2 ` if <something> { bailout }` blocks or a nested check, it may be more code but is still clearer and more expected :)